apps/cms-v2/src/logger-utils/logger-otel-transport.ts

```diff
@@ -0,0 +1,62 @@
+import { ILogObj, Logger } from "tslog";
+import { LogAttributeValue, logs } from "@opentelemetry/api-logs";
+import { context } from "@opentelemetry/api";
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+
+export const attachLoggerOtelTransport = (logger: Logger<ILogObj>) => {
+  logger.attachTransport((log) => {
+    if (!log.payload) {
+      console.error("Logger is not configured properly. OTEL transport will not be attached.");
+
+      return;
+    }
+
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error
```

Let's change it to ts-expect-error, this way it will raise an error if it no longer causes TypeScript to fail